docs/concepts/configuration/taint-and-toleration.md

```diff
@@ -249,9 +249,18 @@ admission controller](https://git.k8s.io/kubernetes/plugin/pkg/admission/default
 
   * `node.alpha.kubernetes.io/unreachable`
   * `node.alpha.kubernetes.io/notReady`
+
+This ensures that DaemonSet pods are never evicted due to these problems,
+which matches the behavior when this feature is disabled.
```

Comment:
Maybe add that the admin can always add other tolerations for ds if he wants

Reply:
There's a line about adding tolerations to arbitrary Pods. I can add explicit mention of DeamonSets, but I'm not sure where and how to phrase it.

Reply:
I think expliciting it here should be just fine, maybe [#49784(comment)](https://github.com/kubernetes/kubernetes/issues/49784#issuecomment-320983833) ?